tests/conftest.py

```diff
@@ -16,9 +16,21 @@
 
 """"Shared fixtures"""
 
+import asyncio
+
+import pytest
 from hexkit.providers.mongodb.testutils import get_mongodb_fixture
-from hexkit.providers.testing.utils import get_event_loop
 
-event_loop = get_event_loop(scope="session")
+
+def event_loop_fixture():
```

Comment:
I cannot comment on this. I think we should avoid depending on a shared event loop anyhow. Not sure why this was introduced. Might be a refactoring goal for later.